{
    "errors": [
        {
            "context": null,
            "message": "Repository already exist.",
            "exceptionName": null
        }
    ]
}